runtime/src/main/java/org/corfudb/runtime/object/CorfuCompileProxy.java

```diff
@@ -81,6 +81,12 @@
     @SuppressWarnings("checkstyle:abbreviation") // Due to deprecation
             UUID streamID;
 
+    /**
+     * The name of the stream of the log.
+     */
+    @Getter
+    String streamName;
```

Comment:
I don't think this is an appropriate place to store the name.
The stream name, which is also the object's name can be exposed through `IStreamView`.

Reply:
I can't change IStreamView to expose a name because it is used in many stream views, whose methods are used by Layout  and Layout is used heavily by NSX clients

Reply:
Not sure I follow. A new method is added to the interface, how could it break other code if its not consumed anywhere?